e2e/commands_test.go

```diff
@@ -367,3 +373,10 @@ func checkContains(t *testing.T, combined string, expectedLines []string) {
 		assert.Assert(t, exp.MatchString(combined), expected, combined)
 	}
 }
+
+func runCmdOnTmpDir(t *testing.T, tmpDir *fs.Dir, command ...string) *icmd.Result {
+	return icmd.RunCmd(icmd.Cmd{
+		Command: command,
+		Dir:     tmpDir.Path(),
+	}).Assert(t, icmd.Success)
```

Comment:
Sometimes we want to check that the [command exited with an error](https://github.com/docker/app/pull/459/files#diff-6d54ca320f3df8927a7543e5ba0f8889R186), it could be useful not to `Assert` here.